core/chain_indexer_test.go

```diff
@@ -51,6 +51,7 @@ func TestChainIndexerSingle(t *testing.T) {
 // Runs multiple tests with randomized parameters and different number of
 // chain backends.
 func TestChainIndexerWithChildren(t *testing.T) {
+	t.Skip("Flaky test from go-ethereum")
```

Comment:
for this again, I'd add some of the context from your PR comment so it's not lost!